ex6/ex6.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Write a string splitter: given a string of comma separated values, would
+ * return an array with the values.
+ *
+ * For example:
+ * split('a,b,c') === ['a', 'b', 'c']
+ * split('hello,world') === ['hello', 'world']
+ * split('hello') === ['hello']
+ * split('hello,,,world') === ['hello', '', '', 'world']
+ * split('') === []
+ *
+ */
+function split(str) {
+
+	var strArray = [], tmpStr = '';
+
+	// If no chars in string no work to do so will return strArray empty array
+	if(str != '') {
```

Comment:
I like to handle these cases explicitly, i.e.:

> if (str === '') {
>   return [];
>}

This leaves the rest of the code clean and without nested if statements.